src/components/pages/RockectDisplay.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable react/forbid-prop-types */
+/* eslint-disable camelcase */
```

Comment:
- [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿

Reply:
Good day @prolajumokeoni, the disabled linters here were for prop-types, it is not accepting array whereas, the prop is an array so I need to use prop-types of array and for it to work I need to disable the eslint that is not allowing it. I won't mind getting views on how to go about it from you. If there is another way to do that, I will like to jump on a zoom call with you for the explanation. Thank you so much.